alcs-frontend/src/app/features/search/application-search-table/application-search-table.component.scss

```diff
@@ -0,0 +1,9 @@
+.spinner-overlay {
```

Comment:
I see this repeated. Given our recent chats about re-usability, could this be made into a shared component?